src/commands/player/FightCommand.js

```diff
@@ -1,13 +1,17 @@
 const Fight = require("../../core/fights/Fight");
 
+module.exports.help = {
+	"name" : "fight"
```

Comment:
Codacy has a fix for the issue: [Extra space after key 'name'.](https://app.codacy.com/gh/DraftBot-A-Discord-Adventure/DraftBot/pullRequest?prid=7575360)
```suggestion
	"name": "fight"
```